app/models/url.rb

```diff
@@ -12,4 +12,11 @@ def generate_slug
   def grab_title
     self.title = TitleHandler.find_title(full_url)
   end
+
+  def self.visits(number)
```

Comment:
Unused method argument - `number`. If it's necessary, use `_` or `_number` as an argument name to indicate that it won't be used. You can also write as `visits(*)` if you want the method to accept any arguments but don't care about them.